app/dockerdwrapper.c

```diff
@@ -361,18 +434,28 @@ start_dockerd(void)
   }
 
   if (use_ipc_socket) {
+    uid_t uid;
+    uid = getuid();
+    uid_t gid;
+    gid = getgid();
```

Comment:
Do you even need these variables, or could you put `getuid()` and `getgit()` in the `snprintf()` call below?